### DISCUSSION TITLE
.m3u download option.

### DISCUSSION THREAD

**Discussion by macron0** [ampache/ampache#2876] [Q&A]
Hi. Am am on ampache 3.8.3.

I have trouble playing music in firefox (very slow on my android 6 tablet)  and I of course want to play with the display off om my phone and just simply use my favorite music player.

I know that selecting stream instead of web player some how sends a .3mu file (at least this works on windows). But I never managed firefox or chrome (on android) to pick it up. I tried saving the windows .m3u  to disk. I put this file on a webpage (linked to it) went to the page with firefox and long pressing the link it let me send the .3mu to an app (tried AIMP and vlc) but than it only plays/sees the first entry. (so I think passing .m3u will never work directly?)  When I long press the link and send it to "download to" (MIX file explorer) to save it to sdcard and than just open it in MIX again which sends it to AIMP it all works fine! 

I tried all the ampache clients for android but none works well. They often just hang after a while. 

If only we could have a "save .m3u" link to long press in my temporary playlist we would have all we need to play music anywhere without hassle. 

The way I do it now is I make a temp playlist on my laptop (windows) and choose stream. Than click play which sends the .3mu and it opens IAMP but I can still find the .3mu that was downloaded. I send this file to my phone via telegram, On my phone I save it to downloads using MIX file explorer and than open the .m3u in MIX that than send it to AIMP and the play list work fine and fast.  But this is soooo much work.

But maybe my post is useless because this option is all ready there in newer version? I could not find any info about it though.

Regards

Hans

**Comment by manuelglez86** [comment]
I have tested you issue on Apache 4.1 and v 5, I think this is because Apache is not designed yet for mobile, maybe you can post this as an issue on the issue section to see if there's something that can be done.
